Doc/library/ssl.rst

```diff
@@ -764,6 +769,15 @@ Constants
 
    .. versionadded:: 3.4
 
+.. data:: OP_NO_TLSv1_3
+
+   Prevents a TLSv1.3 connection. This option is only applicable in conjunction
+   with :const:`PROTOCOL_TLS`. It prevents the peers from choosing TLSv1.3 as
+   the protocol version. TLS 1.3 is available with OpenSSL 1.1.1 or later.
+   With older versions, the flag defaults to *0*.
```

Comment:
Well, other modules don't follow this pattern either, i.e. optional flags are not set to zero when they are not provided by the OS, they are simply not exposed at all. If Twisted wants to be CentOS-compatible, the best thing they can do is to have actual CI on a CentOS system. That's of course complicating things a bit, but it's the only reliable way to reach that goal.